File: dmr-app/src/app/_core/_model/mailing.ts
export interface IMailing {
    id: number ;
    email: string;
    userID: number;
    userName: string;
    frequency: string;
    report: string;
    userNames: string[];
    userIDList: number[];
    userList: any[];
    pathName: string;
    timeSend: Date;
}

File: dmr-api/Models/Mailing.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DMR_API.Models
{
    public class Mailing
    {
        public int ID { get; set; }
        public string Email { get; set; }
        public int UserID { get; set; }
        public string Frequency { get; set; }
        public string Report { get; set; }
        public string PathName { get; set; }
        public DateTime TimeSend { get; set; }
    }
}
